check/controls.go

```diff
@@ -49,7 +51,25 @@ func NewControls(in []byte) (*Controls, error) {
 	// Prepare audit commands
 	for _, group := range c.Groups {
 		for _, check := range group.Checks {
-			check.Commands = textToCommand(check.Audit)
+			if check.SubChecks == nil {
+				check.Commands = textToCommand(check.Audit)
+			} else {
+				for i, SubCheck := range check.SubChecks {
+					check.SubChecks[i].Commands = textToCommand(SubCheck.Audit)
+				}
+			}
+		}
+	}
+
+	if len(definitions) > 0 {
+		c.DefinedConstraints = map[string][]string{}
+		for _, val := range definitions {
+			a := strings.Split(val, "=")
```

Comment:
Should we have an error message if someone passes in something that is a badly-formed constraint? e.g. if someone puts `--define blah` or `--define foo, bar`?